package.json

```diff
@@ -20,6 +20,7 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-react-hooks": "^2.3.0",
```

Comment:
We have to add it to global config. Feel free to make a PR

Reply:
You mean like a peerDependency or what?

Reply:
I mean React hooks support to global configuration if it's not there.

Reply:
It's inherited from airbnb-react, we don't need to configure anything for the time being.